kubernetes.js

```diff
@@ -271,10 +271,10 @@ const createDeployment = async (project, options) => {
         localPod.spec.containers[0].env.push({ name: 'FORGE_LICENSE_TYPE', value: 'ee' })
     }
     if (stack.memory) {
-        localPod.spec.containers[0].env.push({ name: 'FORGE_MEMORY_LIMIT', value: stack.memory })
+        localPod.spec.containers[0].env.push({ name: 'FORGE_MEMORY_LIMIT', value: stack.memory.toString() })
```

Comment:
Does `stack.memory` always return an object (i.e. will toString()` always exist on it?)

Reply:
It should always contain a number, but I suppose we could replace this with 

```suggestion
        localPod.spec.containers[0].env.push({ name: 'FORGE_MEMORY_LIMIT', value: `${stack.memory}` })
```

Reply:
Yep I think that's slightly safer (also a number is a type of object)